README.md

```diff
@@ -63,7 +63,7 @@ Execute command
 
 ```cs
 
-            var connString = $"database={_database};username={_username};password={_password};endpoint={_endpoint};";
+            var connString = $"database={_database};clientid={_clientId};clientsecret={_clientSecret};endpoint={_endpoint};";
```

Comment:
I know it was in the old version of the readme as well, but where does `_variable` come from? I think to eliminate any potential confusion we should just change all those variables to the same. e.g. `clientId`, `clientSecret`, etc.

Reply:
This makes total sense, didn't notice it. Also, there are some more updated to make in Readme, like removing `SetEngine` example since we don't have that function anymore